ilc/server/registry/Registry.js

```diff
@@ -99,10 +99,21 @@ module.exports = class Registry {
         return res.data;
     };
 
-    #getTemplate = async (templateName, locale) => {
+    #getTemplate = async (templateName, locale, domain) => {
```

Comment:
`locale`  can be empty, so potentially we will have the next use case `getTemplate('name', null, 'domain.com')`.
imho it looks awful when it's necessary to provide nullish argument. 
I propose to use one argument as Object - `{locale, domain}`.

Reply:
Updated